lib/utils/Utils.js

```diff
@@ -154,10 +157,11 @@ export default class Utils {
       throw new Error('Expected an array of exactly 1 array element as the JMAP response for a "' + request + '" request.');
     }
 
-    let expectedResponse = Utils._expectedResponseFor(request);
+    let response = data[0][0],
+        expectedResponse = Utils._expectedResponseFor(request);
 
-    if (expectedResponse && data[0][0] !== expectedResponse) {
-      throw new Error('Expected "' + expectedResponse + '" as the JMAP response for a "' + request + '" request, but got "' + data[0][0] + '".');
+    if (response !== ERROR && (expectedResponse && response !== expectedResponse)) {
```

Comment:
There's a test about it. For _forward_ compatibility, we do not want to throw if the backend sends us multiple responses with some responses we do not hande yet, like updates for instance.